pkg/engine/context/loaders/globalcontext.go

```diff
@@ -88,13 +92,26 @@ func (g *gctxLoader) loadGctxData() ([]byte, error) {
 	}
 	g.logger.V(6).Info("variables substituted", "resourcecache", rc)
 
-	storeEntry, ok := g.gctxStore.Get(rc.Name)
+	names := strings.Split(rc.Name, ".")
+	if len(names) < 1 {
+		err := fmt.Errorf("invalid resource cache name %s", rc.Name)
+		g.logger.Error(err, "")
+		return nil, err
+	}
+
+	gctxName := names[0]
+	projectionName := ""
+	if len(names) > 1 {
+		projectionName = names[1]
+	}
```

Comment:
What naming strategy are we following? Is this allowed <projectionName>.<nameA>.<nameB>?